### ISSUE TITLE
Debug mode is not working for Python gauge

### ISSUE THREAD

**Issue by mojito317** [getgauge/gauge-vscode#434]
Hi! I was trying to run my tests with the `Debug Spec` mode, but I always get the following output, however `Run Spec` mode is working fine.
```
Running tool: gauge run .../.spec --simple-console --hide-suggestion
Python: 3.7.6
Runner Ready for Debugging
Run stopped by user.
```

Do I need to make some configurations in my `launch.json` file? I have now this (but it works the exact same way if there is no `launch.json` at all):
```
  // Use IntelliSense to learn about possible attributes.
  // Hover to view descriptions of existing attributes.
  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
  "version": "0.2.0",
  "configurations": [
    {
      "name": "Python: Current File",
      "type": "python",
      "request": "launch",
      "program": "${file}",
      "console": "integratedTerminal"
    }
  ]
}
```

### My environment:
VSCode: Version: 1.37.1
OS: macOS Catalina 10.15.4
Python: 3.7.6
gauge: 1.1.0
getgauge: 0.3.9
Plugins:
```
html-report (4.0.11)
java (0.7.7)
js (2.3.11)
python (0.3.9)
screenshot (0.0.1)
xml-report (0.2.3)
```

**Comment by sriv** [comment]
hi, I am currently on 

```
gauge -v
Gauge version: 1.1.1
Commit Hash: c5323200

Plugins
-------
html-report (4.0.12)
java (0.7.4)
python (0.3.10)
screenshot (0.0.1)
```

and I am using vscode `1.46.1` with gauge-vscode version `0.0.13`.

with the above setup, I am able to debug specs. 

Note: 

- I am using a fresh machine - Linux
- I had trouble with python
  - I had python3 installed, and this had to be configured in the gauge project under `python.properties`
   - I did not have `pip` installed. and I had to install `pip3` and symlink it as `pip`.
